pieFi-platform/hashgraph-names-sdk
# Unstoppable domains

## Describe your changes

1. Integrated Unstoppable Domains Resolution into the Resolver Object
2. Changed the resolveSLD function logic to check for an unstoppable domain name and return that HBAR address instead of returning the result of getSecondLevelDomain.

## Related Tickets

1. Clickup link here
2. Another link if necessary 

## How to test

1. Setup the library and run Resolver.resolveSLD('bensplace.nft') - it should return an HBAR address

## Checklist before requesting a review
- [ X] I have performed a self-review of my code
- [ X] My changes generate no new warningss
- [ X] Any dependent changes have been merged
- [ X] Integration tests pass
- [ X] Linters pass
- [ X] Unit tests pass
